src/main/scala/scala/concurrent/stm/ccstm/Stats.scala

```diff
@@ -49,7 +49,7 @@ private[ccstm] object Stats {
 
     def contents: Seq[Long] = {
```

Comment:
Does it work to simply return a `collection.Seq[Long]` rather than calling `toSeq` below?

Reply:
no, they want an `immutable.Seq` here.

Reply:
Who? Why?

Reply:
It's test data, they are mixing mutable and immutable collection. In 2.12 mutable collection get converted to immutable one. In 2.13 there is a deprecated implicit to let you know this is going on.

Reply:
In 2.12 there is no such conversion. So, the expected type must be `collection.Seq` (not immutable).